lib/python/rose/apps/rose_ana.py

```diff
@@ -101,7 +112,8 @@ def database_lock(self, lockfile, reporter=None):
         if reporter is not None:
             reporter("Acquired DB lock at: " + time.asctime())
         lock.write("{0}".format(os.getpid()))
-        reporter("Writing results to KGO Database...")
+        if reporter is not None:
+            reporter("Writing to KGO Database...")
```

Comment:
No I think this is as intended (I just realised it ought to be protected in the same way the other statements are)